src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/config/SecurityUserContext.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.prisonregister.config
```

Comment:
I copied SecurityUserContext and UserContext from other services...but wondering if I could combine them.  Will do this as a separate PR